crates/astria-conductor/src/executor/mod.rs

```diff
@@ -128,12 +133,29 @@ impl Executor {
             .await
             .wrap_err("executor failed to get commitment state")?;
 
+        // The `executable_block_height` is the height of the sequencer block
+        // where the Conductor sync will be started. The `init_sequencer_height`
```

Comment:
I think it's strictly the sequencer block height which is executable on top of the current rollup state. That sync starts from this height is not the concern of executor, it only cares that it executes the correct block.

Reply:
updated the comment